### ISSUE TITLE
Don't get any work

### ISSUE THREAD

**Issue by slonus** [golemfactory/clay#1700]
Hi I still don't get any tasks. And if I try to send the example task it immediately times out.
![obrazek](https://user-images.githubusercontent.com/19738712/33244788-c9a011c0-d2fd-11e7-9ba2-9b7b9cb410ae.png)
![obrazek](https://user-images.githubusercontent.com/19738712/33244796-d7f56388-d2fd-11e7-9606-3966b98ac6bf.png)
[golem.log](https://github.com/golemfactory/golem/files/1504258/golem.log)

**Comment by badb** [comment]
Hi @slonus!

1) Not getting tasks

Did you change any default options, eg. min. price, reputation?

2) What timeout are you setting with your default task? What kind of scene are using?

**Comment by slonus** [comment]
Hi @badb 

I tried to change both price and reputation, but nothing has happened, so I set back to 0.1.
Also I tried to change the Network trust to 0 - both of them.

For the new task - bmw27_cpu.blend -  I've adjusted my timeout settings to 1 day for task and 10h for subtask and it's now in status waiting.

But no tasks coming from the network.
![obrazek](https://user-images.githubusercontent.com/19738712/33279599-6733807e-d39f-11e7-9764-650fe5b00103.png)
